godot-core/src/builtin/collections/packed_array.rs

```diff
@@ -378,6 +378,27 @@ macro_rules! impl_packed_array {
             pub fn as_inner(&self) -> inner::$Inner<'_> {
                 inner::$Inner::from_outer(self)
             }
+
+            /// Create array filled with default elements.
+            fn default_with_size(n: usize) -> Self {
+                let mut ret = Self::new();
+                ret.resize(n);
+                ret
+            }
+
+            /// Moves elements from slice.
+            ///
+            /// # Safety
+            ///
+            /// * Pointer must be valid and not point to `self` data.
+            /// * Pointer must point to an contiguous array with the same length as `self`.
+            /// * Source data must not be dropped later.
+            unsafe fn move_from_slice(&mut self, src: *const $Element) {
+                // In playground it should optimize to memmove if possible.
+                for (i, v) in self.as_mut_slice().iter_mut().enumerate() {
+                    *v = src.add(i).read();
+                }
```

Comment:
What happened to `drop_in_place` + `copy_nonoverlapping`? Since panics shouldn't happen for builtin types, we could probably use those, no?